Makefile.core.mk

```diff
@@ -33,12 +33,12 @@ export ISTIO_IMAGE_VERSION
 ISTIO_SHA ?= $(shell < ${ISTIOIO_GO}/go.mod grep 'istio.io/istio v' | cut -d'-' -f3)
 export ISTIO_SHA
 
-HUB ?= gcr.io/istio-testing
+HUB ?= gcr.io/istio-prerelease-testing
```

Comment:
these should be overriden by prow and stay wit their current defaults.

Reply:
The issue is when someone tries to run them locally, they don't know they need to override them, so things are broken. This was made so that tests run easily locally and in the CI/CD.

We should revisit this as part of the work for https://github.com/istio/istio.io/issues/7081.

Reply:
Moved back to using daily build repo as it does contain the release builds as well.